add-aws-env.sh

```diff
@@ -0,0 +1,6 @@
+# environment variables used by rust tests directly
+export AWS_ACCESS_KEY_ID=AKIAIOSFODNN7EXAMPLE
+export AWS_SECRET_ACCESS_KEY=wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
+# config file used by aws cli tool
+export AWS_CONFIG_FILE=`pwd`/.aws/config
+RUST_LOG=sim1h=trace
```

Comment:
I don't think we need this file any more as I copied over the nix commands from sim1h. Right, @maackle?